Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/model/ContactDiaryLocation.kt

```diff
@@ -8,6 +8,7 @@ interface ContactDiaryLocation : HasStableId {
     var locationName: String
     val phoneNumber: String?
     val emailAddress: String?
+    val traceLocationGUID: String?
```

Comment:
Please name it `traceLocationID` to match the new requirements from Max

Reply:
It was clearly said by Max "You may keep any variables as "GUID" instead of "ID" if it's too much trouble and the data structures are compatible".
Anyway, I adjusted it